gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -717,3 +728,276 @@ message TranslatedString {
+  optional TranslatedString stop_headsign = 2;
+  // Indicates pickup method.
+  optional PickupType pickup_type = 3 [default = REGULAR_PICKUP];
+  // Indicates drop off method.
+  optional DropOffType drop_off_type = 4 [default = REGULAR_DROP_OFF];
+  // Actual distance traveled along the associated shape, from the first stop to the stop specified in this record.
+  optional float shape_dist_traveled = 5;
+  // Identifies the new serviced stop (defined in the RT-GTFS).
+  optional string stop_id = 6;
+
+  // Valid options for pickup_type:
+  enum PickupType {
+    REGULAR_PICKUP = 0;
+    NO_PICKUP = 1;
+    MUST_PHONE_AGENCY_PICKUP = 2;
+    MUST_ASK_DRIVER_PICKUP = 3;
+  }
+
+  // Valid options for drop_off_type:
+  enum DropOffType {
+    REGULAR_DROP_OFF = 0;
+    NO_DROP_OFF = 1;
+    MUST_PHONE_AGENCY_DROP_OFF = 2;
+    MUST_ASK_DRIVER_DROP_OFF = 3;
+  }
+}
+
+// Describes new trips.
+message Trip {
+  // Identifier of the trip. Must be different than any trip_id defined in the GTFS.
+  required string trip_id = 1;
+  // Identifies a route.
+  optional string route_id = 2;
+  // Text that appears on signage identifying the trip's destination to riders.
+  required TranslatedString trip_headsign = 3;
+  // Public facing text used to identify the trip to riders, for instance, to identify train numbers for commuter rail trip
+  required string trip_short_name = 4;
+  // Indicates the direction of travel for a trip.
+  optional uint32 direction_id = 5;
+  // Identifies the block to which the trip belongs.
+  optional string block_id = 6;
+  // Identifies a geospatial shape describing the vehicle travel path for a trip.
+  optional string shape_id = 7;
+  // Indicates wheelchair accessibility.
+  optional WheelchairAccessibleStatus wheelchair_accessible = 8 [default = UNKNOWN_WHEELCHAIR_ACCESSIBILITY];
+  // Indicates whether bikes are allowed or not.
+  optional BikesAllowedStatus biked_allowed = 9 [default = UNKNOWN_BIKES_ALLOWANCE];
+  //  Defines which trips are being replaced by this trip.
+  repeated string replaces_trip_id = 10;
+
+  // Options for wheelchair_accessible:
+  // 0 - No accessibility information for the trip.
+  // 1 - Vehicle being used on this particular trip can accommodate at least one rider in a wheelchair.
+  // 2 - No riders in wheelchairs can be accommodated on this trip.
+  enum WheelchairAccessibleStatus {
+    UNKNOWN_WHEELCHAIR_ACCESSIBILITY = 0;
+    WHEELCHAIR_ACCESSIBLE = 1;
+    NOT_WHEELCHAIR_ACCESSIBLE = 2;
+  }
+
+  // Options for biked_allowed:
+  // 0 - No bike information for the trip.
+  // 1 - Vehicle being used on this particular trip can accommodate at least one bicycle.
+  // 2 - No bicycles are allowed on this trip.
+  enum BikesAllowedStatus {
+    UNKNOWN_BIKES_ALLOWANCE = 0;
+    BIKES_ALLOWED = 1;
+    BIKES_NOT_ALLOWED = 2;
+  }
+}
+
+// Describes the times that a vehicle arrives at and departs from stops for each updated trip
+message StopTime {
```

Comment:
Is this used anywhere?

Reply:
Thanks for catching this! I believe it should be top-level in `FeedEntity` like `Trip`, `Shape`, etc.

@lionel-nj Could you please take a look and update the PR?

Reply:
I'll take a look at this, thanks

Reply:
After reading the proposal, I confirm that this should be top level in `FeedEntity`. Modyfing this in the next commit.

Reply:
Actually, looking at this more closely, `StopTime` needs to be a `repeated` entity in `Trip`. Otherwise there is no way to tie the `StopTime` to a particular trip. The example is structured like this, but there isn't an entry in the table:
https://docs.google.com/document/d/1oPfQ6Xvui0g3xiy1pNiyu5cWFLhvrTVyvYsMVKGyGWM/edit?pli=1#bookmark=id.v6hpeknwiyjm

I'll make the change shortly in the Google Doc, here, and the bindings.